bin/liquidator/src/rebalance.rs

```diff
@@ -493,6 +509,23 @@ impl Rebalancer {
                     mango_v4::state::SelfTradeBehavior::DecrementTake,
                 )
                 .await?;
+
+            let mut tx_builder = TransactionBuilder {
+                instructions: ixs.to_instructions(),
+                signers: vec![self.mango_client.owner.clone()],
+                ..self.mango_client.transaction_builder().await?
+            };
+
+            let seq_check_ix = self
+                .mango_client
+                .sequence_check_instruction(&self.mango_account_address, account)
+                .await?;
+            tx_builder.append(seq_check_ix);
```

Comment:
minor: In cases where we have prepared instructions (ixs) I think it's preferable to add the seq-check-ix to that, to make its cu budget estimate be included when .to-instructions() is called